pisek/task_jobs/solution.py

```diff
@@ -90,7 +133,7 @@ def _create_batch_jobs(
         )
         run_solution.add_prerequisite(self._compile_job)
 
-        if sub_num == 0:
+        if subtask == 0:
```

Comment:
Why is subtask 0 treated differently here? Shouldn't the important thing be if it's static or not?